doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -119,8 +119,9 @@ For lists of protocol-specific changes, see `Protocols`_.
 Bluetooth samples
 -----------------
 
-|no_changes_yet_note|
+* :ref:`ble_throughput` sample:
 
+  * Fixed peer throughput calculations.
```

Comment:
Should we elaborate this a bit? How these calculations were wrong or misleading before the fix?

Reply:
I'd suggest:
```
Fixed peer throughput calculations.
These were too low because the total transfer time incorrectly included 500ms delay without including the actual transfer.
```

Reply:
Done